ui/src/routes/about_us.tsx

```diff
@@ -381,11 +394,17 @@ function AboutUs() {
                 );
               })}
             </Row>
-            <div className="atc-title">
+            .{" "}
```

Comment:
do we need the `.`?

Reply:
nope, that shouldn't be here - I remove it